framework/console/widgets/Table.php

```diff
@@ -374,14 +374,8 @@ protected function getScreenWidth()
     {
         if (!$this->_screenWidth) {
             $size = Console::getScreenSize();
-            if (isset($size[0])) {
-                $this->_screenWidth = $size[0];
-            }
-            else {
-                $this->_screenWidth = 123;
-            }
+            $this->_screenWidth = isset($size[0]) ? $size[0] : 123;
```

Comment:
Thanks!
 I am trying to take out magic numbers (123). Maybe would be better make a constant like DEFAULT_CONSOLE_WIDTH = 120 and make expression like 
~~~php
$this->_screenWidth = isset($size[0]) ? $size[0] : self::DEFAULT_CONSOLE_WIDTH + 3;
~~~
where new magic number should also take out to constant TERMINAL_SCROLLBAR_WIDTH(i am not sure what it is)

Reply:
@dmirogin Agree with you.